packages/api/app/Utils/cache.js

```diff
@@ -1,26 +1,53 @@
+/** @type {import('ioredis').Redis} */
 const Redis = use('Redis');
 
-const get = async (key = '') => {
+async function get(key = '') {
 	const result = await Redis.get(key);
 	return result ? JSON.parse(result) : false;
-};
+}
 
-const set = async (key = '', value = '', expires = 60, ...rest) => {
+async function set(key = '', value = '', expires = 60, ...rest) {
 	return Redis.set(key, JSON.stringify(value), 'EX', expires, ...rest);
-};
+}
+
+async function keys(pattern) {
+	return Redis.keys(pattern);
+}
 
-const keys = async (...params) => Redis.keys(...params);
+async function del(...params) {
+	return Redis.del(...params);
+}
+
+/**
+ * Deletes keys by prefix
+ *
+ * @param {string} keyPrefix Redis key prefix
+ */
+async function deleteByPrefix(keyPrefix) {
```

Comment:
Now we can invalidate the cache using the key prefix! 🎉 
![partyparrt-30](https://user-images.githubusercontent.com/30202634/118207273-b2744800-b43a-11eb-8438-669e0353d83f.gif)